dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,138 @@
+﻿import "oaidl.idl";
+import "ocidl.idl";
+
+[contract(AppNotificationsContract, 1)]
+runtimeclass AppNotificationButton
```

Comment:
Missing in AppNotificationButton: buttons with colors. See https://docs.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/adaptive-interactive-toasts?tabs=builder-syntax#buttons-with-colors